testutils/test_server.go

```diff
@@ -60,15 +60,17 @@ type TestServer struct {
 	// as part of the TestServer (including the server).
 	channelStates map[*tchannel.Channel]*tchannel.RuntimeState
 
-	verifyOpts *goroutines.VerifyOpts
-	postFns    []func()
+	introspectOpts *tchannel.IntrospectionOptions
+	verifyOpts     *goroutines.VerifyOpts
+	postFns        []func()
 }
 
 // NewTestServer constructs a TestServer.
 func NewTestServer(t testing.TB, opts *ChannelOpts) *TestServer {
 	ts := &TestServer{
-		TB:            t,
-		channelStates: make(map[*tchannel.Channel]*tchannel.RuntimeState),
+		TB:             t,
+		channelStates:  make(map[*tchannel.Channel]*tchannel.RuntimeState),
+		introspectOpts: &tchannel.IntrospectionOptions{IncludeExchanges: true},
```

Comment:
IncludeTombstones by default, if the test fails, we want to see all the relay items